controllers/provisioners/ekscloudformation/ekscloudformation.go

```diff
@@ -488,3 +489,24 @@ func (ctx *EksCfInstanceGroupContext) processParameters() error {
 	ctx.parseTags()
 	return nil
 }
+
+//getManagedPolicyARNs constructs managed policy arns
+func getManagedPolicyARNs(pNames []string) string {
+	//This is for Managed Policy ARN list.
+	//First add the DEFAULT required policies to the list passed by user as part of custom resource
+	var managedPolicyARNs []string
+	requiredPolicies := []string{"AmazonEKSWorkerNodePolicy", "AmazonEKS_CNI_Policy", "AmazonEC2ContainerRegistryReadOnly"}
+
+	for _, name := range requiredPolicies {
+		managedPolicyARNs = append(managedPolicyARNs, fmt.Sprintf("arn:aws:iam::aws:policy/%s", name))
+	}
+	// Add the user supplied policy names if any
+	if len(pNames) != 0 {
+		for _, name := range pNames {
+			// Trim the prefix arn:aws:iam::aws:policy/ if user supplied entire ARN instead of just name
+			managedPolicyARNs = append(managedPolicyARNs, fmt.Sprintf("arn:aws:iam::aws:policy/%s", strings.TrimPrefix(name, "arn:aws:iam::aws:policy/")))
+		}
```

Comment:
Nice,
minor nit: consider improving this so that
```
prefix := "arn:aws:iam::aws:policy/"
```

That would make this statement a bit more readable:
```
prefix := "arn:aws:iam::aws:policy/"
managedPolicyARN := fmt.Sprintf("%v%v", prefix, strings.TrimPrefix(name, prefix))
managedPolicyARNs = append(managedPolicyARNs, managedPolicyARN)
```